projects/notebooks/testing/generate_matrix-benchmarking.sh

```diff
@@ -21,309 +21,310 @@ export MATBENCH_WORKLOAD=projects.notebooks.visualizations.$(get_config matbench
 WORKLOAD_STORAGE_DIR="$(echo "$MATBENCH_WORKLOAD" | tr . /)"
 
 if [[ "${JOB_NAME_SAFE:-}" == "plot" ]]; then
-    set_config_from_pr_arg 1 "matbench.preset"
+	set_config_from_pr_arg 1 "matbench.preset"
 fi
 
 matbench_preset=$(get_config matbench.preset)
 
 if [[ "$matbench_preset" == null ]]; then
-    # no preset defined
-    true
+	# no preset defined
+	true
 
 elif [[ "$matbench_preset" == "https://"* ]]; then
-    set_config matbench.download.url "$matbench_preset"
+	set_config matbench.download.url "$matbench_preset"
 else
-    set_config matbench.config_file "${matbench_preset}.yaml"
-    set_config matbench.download.url_file "${WORKLOAD_STORAGE_DIR}/data/${matbench_preset}.yaml"
+	set_config matbench.config_file "${matbench_preset}.yaml"
+	set_config matbench.download.url_file "${WORKLOAD_STORAGE_DIR}/data/${matbench_preset}.yaml"
 fi
 
 get_matbench_config() {
-    CI_ARTIFACTS_FROM_CONFIG_FILE=$WORKLOAD_STORAGE_DIR/data/$(get_config matbench.config_file) \
-        get_config "$@"
+	CI_ARTIFACTS_FROM_CONFIG_FILE=$WORKLOAD_STORAGE_DIR/data/$(get_config matbench.config_file) \
+		get_config "$@"
 }
 
```

Comment:
in this file, could you revert the indentation to 4 spaces, and add the few empty lines back? (or are these empty lines inconsistent with the rest of the file? 🤔 )